### DISCUSSION TITLE
v4.3.4

### DISCUSSION THREAD

**Discussion by JastinXyz** [Mengkodingan/ckptw#18] [Announcements]
## changes

- bump `@whiskeysockets/baileys` to v6.7.7
- add `WAVersion` options in client constructor.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mengkodingan/ckptw/releases/tag/v4.3.4'>v4.3.4</a>.</em>

**Comment by Cyberkingcr7** [comment]
I think you should also add logger
